pandas/io/json/table_schema.py

```diff
@@ -6,7 +6,7 @@
 import warnings
 
 import pandas._libs.json as json
-from pandas import DataFrame
+from pandas import DataFrame, Timedelta
```

Comment:
Import `to_timedelta` instead

Reply:
Oh, sorry, I've pushed this accidentally